src/transformers/file_utils.py

```diff
@@ -28,10 +27,10 @@
 from filelock import FileLock
 from tqdm.auto import tqdm
 
-from . import __version__
+from . import __version__, hf_logging
 
 
-logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+logger = hf_logging.get_logger()  # pylint: disable=invalid-name
```

Comment:
It would be nice to keep some detail of the filename in which the log happened (cc @lhoestq).
Can we keep `__name__` as an argument here?